contracts/StreamManagerStorageV1.sol

```diff
@@ -0,0 +1,29 @@
+//SPDX-License-Identifier: Unlicense
+pragma solidity ^0.8.0;
+
+contract StreamManagerStorageV1 {
+    struct OpenStream {
+        address payee;
+        address token;
+        uint256 rate;
+        uint256 terminationPeriod;
+        uint256 cliffPeriod;
+        uint256 createdAt;
+        uint256 lastClaimedAt;
+        uint256 terminatedAt;
+        bool isTerminated;
+    }
+
+    ///@dev admin address
+    address public admin;
+    ///@dev payer address
+    address public payer;
+    /// @dev payee's address => instance
+    mapping(address => OpenStream) public streamInstances;
+    /// @dev payee's address => true/false
+    mapping(address => bool) public isPayee;
+
+    uint256[64] private __gap__;
```

Comment:
@call3hudson Can you explain why do you put that value?

Reply:
The gap is needed for adding extra states in upgradeable proxies.
If we have to add a new state storage in the next version, the size of the gap needs to be updated appropriately.

Reply:
But how about the size '46' as we have 4 slots already occupied?

Reply:
That's good, will fix it